### DISCUSSION TITLE
Copilot Labs: Need to accept Pre-Release License Terms

### DISCUSSION THREAD

**Discussion by tricki** [community/community#48120] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I got the following message in VS Code: 

> To use GitHub Copilot Labs, you need to accept the GitHub Next Pre-Release License Terms. Sign up by visiting https://githubnext.com/projects/copilot-labs/, then restart Visual Studio Code.

Where on that page can I accept the terms? It took me a while just to find them (at https://github.com/githubnext/githubnext/blob/main/TERMS_AND_CONDITIONS.md) but it's a simple GitHub markdown file with no option to accept.

Are those the correct terms? Do I need to sign up somewhere or accept the terms? And how/where can I accept them?

**Comment by arcolife** [comment, marked as answer]
@tricki wasn't obvious to me at first either since there's no page anchor defined for specifically accepting terms. You gotta click sign up under https://githubnext.com/projects/copilot-labs/#how-to-use-copilot-labs

**Reply by tricki** [reply]
A link that doesn't look like all other links and doesn't even have an `[href]` so it looks like it just links to the same page 🤦. Thanks for the help.

**Reply by albertumus** [reply]
Hi! I'm checking this answer but it looks like the URL for Sign up is broken :(

**Reply by filonfighter** [reply]
Unfortunately I still cannot find the sign up and can't access copilot labs. Can you give me some advice how to accept the terms?

**Reply by huchiuan** [reply]
@filonfighter  I have same question , Any body know the main point?

**Reply by Senthilsteve** [reply]
Github has retried the preview of Copilot Labs, Hence the copilot labs status will be completed ( [check here](https://githubnext.com/projects/copilot-labs/#how-to-use-copilot-labs) ) and there won't be any option to accept the terms (it will be only available if the extension is in preview or development state)

The solution will be to uninstall Copilot labs and install the new [Copilot chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat)

**Comment by EdwinFairchild** [comment]
Im also seeing this issue, Copilot works, Copilot chat works, but Copilot labs keep giving me an authentication error, claims to click. a link and there will be a sign up page and it does not exist. Like in the answer above takes me to the same page the extension takes me to, where there is no sign up link. 
It works one in a blue, but 95% of the time I get this error.

**Comment by GabrielVeres95** [comment]
I am having the same issues. There is no Sign Up on the page

**Comment by adisuyash** [comment]
Can someone please resolve this issue :
![image](https://github.com/community/community/assets/116362593/28e18dbd-5b1a-42d2-a8f5-625492e32733)

**Reply by linzhaolover** [reply]
I have the same issue.

**Comment by filonfighter** [comment]
Got answer;)
![image](https://github.com/community/community/assets/96781387/7c1599e8-f082-462e-aa91-7d537a3e4b0b)

**Comment by adisuyash** [comment]
Everyone there's a **catch !**
I can use **GitHub Copilot** without any issue in **VSCODE** 
That notification seems quite useless for me :)

**Reply by EdwinFairchild** [reply]
this was not about Copilot this was about Copilot Labs

**Reply by adisuyash** [reply]
Ohh My fault :(
